internal/server/data/schema/parse.go

```diff
@@ -23,6 +23,7 @@ func ParseSchema(schema string) ([]Statement, error) {
 	var currentTable string
 	var currentStmt strings.Builder
 	var stmts []Statement
+	var inCodeBlock bool
```

Comment:
Shouldn't `inCodeBlock` be part of the `state` switch, instead of a separate sub-state